python/cudf/cudf/_lib/copying.pyx

```diff
@@ -141,14 +150,16 @@ def gather(Table source_table, Column gather_map, bool keep_index=True):
     else:
         source_table_view = source_table.data_view()
     cdef column_view gather_map_view = gather_map.view()
-    cdef bool c_bounds_check = True
+    cdef cpp_copying.out_of_bounds_policy policy = (cpp_copying
+                                                    .out_of_bounds_policy
+                                                    .DONT_CHECK)
```

Comment:
Can't wait until we have `black` for Cython so that we don't have to debate style, but I'd prefer:

```suggestion
    cdef cpp_copying.out_of_bounds_policy policy = (
        cpp_copying.out_of_bounds_policy.DONT_CHECK
    )
```

The current style seems more suited for method chaining.